padertorch/contrib/examples/vae_mnist/mnist_vae_example.py

```diff
@@ -86,39 +94,38 @@ def review(self, inputs, outputs):
 class VAE(nn.Module):
     """ A simple Variational Autoencoder"""
 
-    def __init__(self):
-        super(VAE, self).__init__()
-        self.kernel_size = 4
-
+    def __init__(self, activation_function, hidden_channels):
+        super().__init__()
+        kernel_size = 4
```

Comment:
why is the kernel size not an input to the init?

Reply:
should I add an additional condition for the kernel size here? with three layers, the kernel size cannot be arbitrarily large

Reply:
You could add a maximum size, however why shouldnt one use a kernel size of 3 or 5?